engine/src/main/java/org/hibernate/validator/internal/util/Version.java

```diff
@@ -17,15 +14,9 @@
  * @author Kevin Pollet &lt;kevin.pollet@serli.com&gt; (C) 2012 SERLI
  */
 public final class Version {
-	private static final Pattern JAVA_VERSION_PATTERN = Pattern.compile( "^(?:1\\.)?(\\d+)$" );
```

Comment:
Mmmmh, why do we remove all this? I think we should keep it. We probably will have version specific features in the future and I don't see the point in removing this?